### ISSUE TITLE
Previous response returned multiple times

### ISSUE THREAD

**Issue by esledov** [terry3041/pyChatGPT#108]
I am running multiple prompts and collecting responses within the same conversation

(This is an illustrative example)

`for i in range(5):
    prompt = get_new_prompt()
    resp = self.api.send_message(prompt)
    result = resp['message']`

On multiple occasions, I see that the returned response is not for the current prompt, but for the previous one.

prompt1: response for prompt1
prompt2: response for prompt1
prompt3: response for prompt1
prompt4: response for prompt4

Basically, I am getting correct response for prompt1, but self.api.send_message(prompt2) returns the same response. And this streak sometimes goes for 2-10 prompts. Then again, I am getting a single correct response, and it is returned for several more new prompts.

Is there any way to make sure that I am getting the correct response for a given prompt every time? I assume it is returning the last response from many responses on the page and is not always getting updated?

Please?

**Comment by terry3041** [comment]
Check v0.4.3.3 and see if it works.

**Comment by esledov** [comment]
No, I am still getting the same error.

**Comment by CodeBlog-x64** [comment]
yeah im having the same exact issue too. This never happened before but as of the last 2-3 days it's been happening every few prompts.
